src/scripts/main.js

```diff
@@ -1,3 +1,57 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tbody = table.querySelector('tbody');
```

Comment:
Ensure that the table and tbody elements exist in the DOM before querying them to avoid potential errors if the script runs before the DOM is fully loaded.